test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -497,7 +497,7 @@ Function sys.objectpropertyex(integer,sys."varchar")
 Function sys.openjson_array(text)
 Function sys.openjson_object(text)
 Function sys.openjson_with(text,text,text[])
-Function sys.openquery(text,text)
+Function sys.openquery_internal(text,text)
```

Comment:
why is this coming here? due to openquery_internal not being in older releases?

Reply:
Yes sys.openquery_internal() is now replacing sys.openquery() hence the diff